backend/prisma/schema.prisma

```diff
@@ -32,8 +50,100 @@ model Session {
 }
 
 model Profile {
-  id     String @id @default(uuid())
-  user   User @relation(fields: [userId], references: [id])
-  userId String @unique @default(uuid())
-  photoUrl String? 
-}
\ No newline at end of file
+  profileID String  @id @default(uuid())
+  user      User    @relation(fields: [userId], references: [UserID])
+  userId    String  @unique
+  photoUrl  String?
+}
+
+model Plan {
+  PlanID           String      @id @default(uuid())
+  description      String
+  startDate        DateTime
+  endDate          DateTime
+  createdDate      DateTime    @default(now())
+  specialEventType EventType
+  contact          Contact?    @relation(fields: [contactID], references: [ContactID])
+  contactID        String?     @unique
+  User             Pledge[]
+  FinalGift        FinalGift?
+}
+
+model Pledge {
+  PledgeID     String        @id @default(uuid())
+  plan         Plan          @relation(fields: [planID], references: [PlanID])
+  planID       String
+  user         User          @relation(fields: [userID], references: [UserID])
+  userID       String
+  inviteStatus InviteStatus?
+  role         Role
+  giftChoice   GiftChoice[]
+}
+
+enum Role {
+  PLAN_LEADER
+  FRIEND
+}
+
+enum InviteStatus {
+  INVITED
+  RESENT_INVITE
+  ACCEPTED
+  DENY
+  NOT_APPLICABLE
+}
+
+model Contact {
+  ContactID          String                    @id @default(uuid())
+  email              String?
+  firstName          String
+  lastName           String?
+  phoneNumber        String?
+  address            String
```

Comment:
maybe we omit this for now since fahad is punting on address for now - he should be migrating this field onto the Contact table separately once he figures out the implementation there